Resources/Prototypes/Catalog/Fills/Lockers/heads.yml

```diff
@@ -1,308 +1,395 @@
+    - id: ClothingNeckGoldmedal
+    - id: DoorRemoteService
+    - id: HoPIDCard
+    - id: IDComputerCircuitboard
+    - id: RubberStampApproved
+    - id: RubberStampDenied
+    - id: RubberStampHop
+    - id: WeaponDisabler
 
 - type: entity
   id: LockerHeadOfPersonnelFilled
   suffix: Filled
   parent: LockerHeadOfPersonnel
   components:
-  - type: StorageFill
-    contents:
-      - id: HoPIDCard
-      - id: ClothingHeadsetCommand
-      - id: BoxPDA
-      - id: BoxID
-      - id: BoxHeadset
-      - id: IDComputerCircuitboard
-      - id: WeaponDisabler
-      - id: CigarGoldCase
-        prob: 0.25
-        # Fuck the HoP they don't deserve fucking cigars.
-        # Yes they do fuck you.
-      - id: DoorRemoteService
-      - id: ClothingNeckGoldmedal
-      - id: RubberStampHop
-      - id: RubberStampDenied
-      - id: RubberStampApproved
-      - id: BoxEncryptionKeyPassenger
-      - id: BoxEncryptionKeyService
-      - id: ClothingBackpackIan
-        prob: 0.5
-      - id: AccessConfigurator
+  - type: EntityTableContainerFill
+    containers:
+      entity_storage: !type:NestedSelector
+        tableId: LockerFillHeadOfPersonnel
 
-- type: entity
-  id: LockerChiefEngineerFilledHardsuit
-  suffix: Filled, Hardsuit
-  parent: LockerChiefEngineer
-  components:
-  - type: StorageFill
-    contents:
-      - id: ClothingOuterHardsuitEngineeringWhite
-      - id: ClothingMaskBreath
-      - id: ClothingEyesGlassesMeson
-      - id: ClothingBeltChiefEngineerFilled
-      - id: ClothingShoesBootsMagAdv
-      - id: ClothingHandsGlovesColorYellow
-      - id: CigarCase
-        prob: 0.15
-      - id: DoorRemoteEngineering
-      - id: RubberStampCE
-      - id: JetpackVoidFilled
-      - id: ClothingHeadsetAltEngineering
-      - id: BoxEncryptionKeyEngineering
-      - id: AccessConfigurator
-      - id: RCD
-      - id: RCDAmmo
+# No hardsuit table
+- type: entityTable
+  id: LockerFillChiefEngineerNoHardsuit
+  table: !type:AllSelector
+    children:
+    - id: AccessConfigurator
+    - id: BoxEncryptionKeyEngineering
+    - id: CigarCase
+      prob: 0.15
+    - id: ClothingBeltChiefEngineerFilled
+    - id: ClothingEyesGlassesMeson
+    - id: ClothingHandsGlovesColorYellow
+    - id: ClothingHeadsetAltEngineering
+    - id: DoorRemoteEngineering
+    - id: RCD
+    - id: RCDAmmo
+    - id: RubberStampCE
+
+# Hardsuit table, used for suit storage as well
+- type: entityTable
+  id: FillChiefEngineerHardsuit
+  table: !type:AllSelector
+    children:
+    - id: ClothingMaskBreath
+    - id: ClothingOuterHardsuitEngineeringWhite
+    - id: ClothingShoesBootsMagAdv
+    - id: JetpackVoidFilled
+    - id: OxygenTankFilled
+
+# No hardsuit table + hardsuit table
+- type: entityTable
+  id: LockerFillChiefEngineerHardsuit
+  table: !type:AllSelector
+    children:
+    - !type:NestedSelector
+      tableId: LockerFillChiefEngineerNoHardsuit
+    - !type:NestedSelector
+      tableId: FillChiefEngineerHardsuit
```

Comment:
for things like this, you can just put this directly on the prototype instead of making a separate table prototype. It's not going to be reused so at this point its just extra lines.
```yml
- type: EntityTableContainerFill
    containers:
      entity_storage: !type:AllSelector
        children:
        - !type:NestedSelector
          tableId: LockerFillChiefEngineerNoHardsuit
        - !type:NestedSelector
          tableId: FillChiefEngineerHardsuit
```

it would probably reduce the line count even more.